backon/src/backoff/constant.rs

```diff
@@ -36,6 +36,7 @@ pub struct ConstantBuilder {
     delay: Duration,
     max_times: Option<usize>,
     jitter: bool,
+    seed: u64,
```

Comment:
Hi, please store `None<u64>` instead so we can know if users set their own seed.